modules/dataquery/ajax/datadictionary.php

```diff
@@ -19,16 +19,33 @@
 $client = new NDB_Client();
 $client->makeCommandLine();
 $client->initialize(__DIR__ . "/../../../project/config.xml");
-$cdb      = CouchDB::singleton();
-$category = $_REQUEST['category'];
-$results  = $cdb->queryView(
-    "DQG-2.0",
-    "datadictionary",
-    array(
-     "reduce"   => "false",
-     "startkey" => "[\"$category\"]",
-     "endkey"   => "[\"$category\", \"ZZZZZZZZ\"]",
-    )
-);
+
+$cdb = CouchDB::singleton();
+if ($_REQUEST['category']) {
+    $category = urlencode($_REQUEST['category']);
+
+    $results = $cdb->queryView(
+        "DQG-2.0",
+        "datadictionary",
+        array(
+         "reduce"   => "false",
+         "startkey" => "[\"$category\"]",
+         "endkey"   => "[\"$category\", \"ZZZZZZZZ\"]",
```

Comment:
I'm not quite sure, this is just legacy code that I didnt touch just indented. If this is the case I think this should be added to a separate pull request.